.ci/setup_env_centos.sh

```diff
@@ -65,7 +65,7 @@ declare -A packages=( \
 	[crio_dependencies]="libassuan-devel libgpg-error-devel device-mapper-libs util-linux libselinux-devel" \
 	[bison_binary]="bison" \
 	[libgudev1-dev]="libgudev1-devel" \
-	[general_dependencies]="gpgme-devel glib2-devel glibc-devel bzip2 m4 gettext-devel automake autoconf pixman-devel coreutils" \
+	[general_dependencies]="xfsprogs gpgme-devel glib2-devel glibc-devel bzip2 m4 gettext-devel automake autoconf pixman-devel coreutils" \
```

Comment:
These lines are getting rather long for my liking. We should fold them now/soon (and sort alphabetically) I think.

Reply:
yes, maybe I can do that in another PR list all them alphabetically for all distros